src/typography/src/Code.js

```diff
@@ -1,21 +1,29 @@
 import React, { forwardRef, memo } from 'react'
 import PropTypes from 'prop-types'
-import useCodeAppearance from '../../theme/src/hooks/useCodeAppearance'
+import useStyleConfig from '../../hooks/use-style-config'
 import Text from './Text'
 
+const pseudoSelectors = {}
+const internalStyles = {}
+
 const Code = memo(
   forwardRef(function Code(props, ref) {
     const { className, appearance = 'default', ...restProps } = props
 
-    const { boxProps } = useCodeAppearance({ appearance })
+    const styleProps = useStyleConfig(
+      'Code',
+      { appearance },
```

Comment:
This really should pass `props` in entirety but we also need the default value somehow... I think we could define the theme defaults alongside the component theming object itself?